e2e/snapshot-creators/default.cy.snap.js

```diff
@@ -274,3 +280,29 @@ describe("snapshots", () => {
     });
   });
 });
+
+function getDefaultInstanceData() {
+  const instanceData = {};
+
+  cy.request("/api/card").then(({ body: cards }) => {
+    instanceData.questions = cards;
+  });
+
+  cy.request("/api/dashboard").then(({ body: dashboards }) => {
+    instanceData.dashboards = dashboards;
+  });
+
+  cy.request("/api/user").then(({ body: { data: users } }) => {
+    instanceData.users = users;
+  });
+
+  cy.request("/api/database").then(({ body: { data: databases } }) => {
+    instanceData.databases = databases;
+  });
+
+  cy.request("/api/collection").then(({ body: collections }) => {
+    instanceData.collections = collections;
+  });
+
+  return instanceData;
+}
```

Comment:
this is the meat of this whole thing. We could easily add any other instance data by just shoving it into the object to get cached on test start.